docs/language/operators/merge.md

```diff
@@ -9,14 +9,14 @@
 ```
 ### Description
 
-The `merge` operator merges inputs from multiple upstream legs of
-the dataflow path into a single output.  The order of values in the combined
+The `merge` operator merges inputs from multiple upstream branches of
+the pipeline into a single output.  The order of values in the combined
 output is determined by the `<expr>` arguments, which act as sort expressions
-where the values from the upstream paths are forwarded based on these expressions.
+where the values from the upstream pipeline branches are forwarded based on these expressions.
 
 ### Examples
 
-_Copy input to two paths and combine_
+_Copy input to two pipeline branches and merge_
```

Comment:
Nit: Make the same change you made to the similar example in docs/language/operators/fork.md.